nuitka/build/static_src/HelpersFilesystemPaths.c

```diff
@@ -889,15 +899,18 @@ bool expandTemplatePath(char *target, char const *source, size_t buffer_size) {
                 appendStringSafe(target, pid_buffer, buffer_size);
             } else if (strcasecmp(var_name, "HOME") == 0) {
                 char const *home_path = getenv("HOME");
-
                 if (home_path == NULL) {
+#ifdef __wasi__
```

Comment:
That's going to cause issues with much of standalone, etc. but I guess, that's not a topic for that platform?